src/gazebo_camera_manager_plugin.cpp

```diff
@@ -720,25 +714,22 @@ void CameraManagerPlugin::_handle_request_video_stream_information(const mavlink
     // ACK command received and accepted
     _send_cmd_ack(pMsg->sysid, pMsg->compid, MAV_CMD_REQUEST_VIDEO_STREAM_INFORMATION, MAV_RESULT_ACCEPTED, srcaddr);
 
+    mavlink_video_stream_information_t video_stream_information{};
+    video_stream_information.stream_id = 1;
+    video_stream_information.count = 1;
+    video_stream_information.type = VIDEO_STREAM_TYPE_RTPUDP;
+    video_stream_information.flags = VIDEO_STREAM_STATUS_FLAGS_RUNNING; // It's always running
+    video_stream_information.framerate = 30;
+    video_stream_information.resolution_h = _width;
+    video_stream_information.resolution_v = _height;
+    video_stream_information.bitrate = 2048;
+    video_stream_information.rotation = 0; // none
+    video_stream_information.hfov = 90; // made up
+    std::strcpy(video_stream_information.name, "Visual Spectrum");
+    std::strcpy(video_stream_information.uri, _videoURI.c_str());
+
```

Comment:
Note, there is a new encoding field, which ideally you should set to [VIDEO_STREAM_ENCODING_H264](https://mavlink.io/en/messages/common.html#VIDEO_STREAM_ENCODING_H264). If you don't it won't matter because the default value 0 means "don't know" and the recipient should detect. This is to reduce effort on recipient as cameras start to encode in H265.

![image](https://github.com/user-attachments/assets/d9b9b3d0-bede-4e83-88f8-755c3d7dd140)

Reply:
You're probably right, didn't even check what should be set in the extension 😇
I added it in 357396f8c048a8ceadb14cfc6409817b2079d15b

This will probably make CI fail again because the extension does not yet exist on PX4 main until https://github.com/PX4/PX4-Autopilot/pull/23490 is merged...